client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -90,6 +85,27 @@ export const ScheduleForm = ( props: Props ) => {
 		timestamp: validateTimeSlot( { frequency, timestamp }, scheduledTimeSlots ),
 	} );
 	const [ fieldTouched, setFieldTouched ] = useState< Record< string, boolean > >( {} );
+	const trackEventProps = {
+		site_slug: siteSlug,
+		frequency,
+		hour,
+		period,
+		day,
+	};
+	const { createUpdateSchedule } = useCreateUpdateScheduleMutation( siteSlug, {
+		onSuccess: () => {
+			recordTracksEvent( 'calypso_update_manager_schedule_create', trackEventProps );
+
+			return onSyncSuccess && onSyncSuccess();
+		},
+	} );
+	const { editUpdateSchedule } = useEditUpdateScheduleMutation( siteSlug, {
+		onSuccess: () => {
+			recordTracksEvent( 'calypso_update_manager_schedule_edit', trackEventProps );
+
+			return onSyncSuccess && onSyncSuccess();
+		},
+	} );
```

Comment:
In the feature, there is a chance to integrate this form in a different context (ex. multiple sites).
The form will stay the same, but the tracking event should be different.

I suggest moving this `recordTracksEvent` trigger to the parent components, `schedule-create` and `schedule-edit`. It's an easy update; there is already an `onSyncSuccess` callback param.

What do you think?

Reply:
Make sense, thanks.,